app/konnect/privatelink/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Connect your Data Plane to Konnect Control Plane over PrivateLink
+subtitle: Sync configurations privately to stay compliant and save data transfer costs
+content-type: reference
+---
+## Steps
+
+<ol>
+
+<li>
+Connect the AWS Console to your <b>data plane region</b> and create a VPC endpoint.
+
+<center>
+<img src="/assets/images/docs/privatelink-1.png" alt="" style="width: 600px; margin-top:10px; margin-bottom:10px">
+</center>
+</li>
+
+<li>
+Include the Konnect <b>control plane geo</b> <i>(for example: US)</i> you want to connect to in the endpoint name. In this example, we are connecting to Konnect in US geo. Next, choose <i>Other Endpoint Services</i> under Service Category.
+
+<center>
+<img src="/assets/images/docs/privatelink-2.png" alt="" style="width: 600px; margin-top:10px; margin-bottom:10px">
+</center>
+</li>
+
+<li>
+Choose your data plane region from the menu below.
```

Comment:
@healthy-pod I saw [this](https://kongstrong.slack.com/archives/C069WCHF3PW/p1724696155399529) on Slack, do we need to add a tab for `eu-west-1`?